src/injectdll/channel.py

```diff
@@ -37,7 +67,7 @@ def connect(self, n_attempts=30, delay=1):
                 break
             except pywintypes.error as e:
                 if e.args[0] == winerror.ERROR_FILE_NOT_FOUND:
-                    ActionLogger().log('Attempt {}/{}: failed to connect to the pipe {}'.format(i+1, n_attempts,
+                    ActionLogger().log('Attempt {}/{}: failed to connect to the pipe {}'.format(i + 1, n_attempts,
```

Comment:
Since we try to minimize dependencies, I suggest to use just `logging` module. And since this is a library, please don't create a logger here. An application or another library (user) should actually create/enable a logger. We just need to get the logger with the name of the library and use it.